### PR TITLE
Create strings pkg to fix util issue

### PR DESCRIPTION
**Pull request by resouer**
ref #15634 

Use strings pkg to hold strings related tools

### CODE REVIEW

File: pkg/util/strings/strings.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	}
+	return parts[0], parts[1]
+}
+
+// Joins 'namespace' and 'name' and returns a fully qualified name
+// Assumes that the input is valid.
+func JoinQualifiedName(namespace, name string) string {
+	return path.Join(namespace, name)
+}
+func ShortenString(str string, n int) string {
```
Comment (thockin) [inline pkg/util/strings/strings.go]: needs a blank line between, and a comment?

File: pkg/volume/aws_ebs/aws_ebs.go [hunk 2]
```diff
@@ -29,9 +29,9 @@ import (
 	"k8s.io/kubernetes/pkg/api/resource"
 	awscloud "k8s.io/kubernetes/pkg/cloudprovider/providers/aws"
 	"k8s.io/kubernetes/pkg/types"
-	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/util/exec"
 	"k8s.io/kubernetes/pkg/util/mount"
+	stringtool "k8s.io/kubernetes/pkg/util/strings"
```
Comment (thockin) [inline pkg/volume/aws_ebs/aws_ebs.go]: should be utilstrings

**Comment by resouer** [comment]
@thockin  Thanks for quick response, just updated

**Comment by thockin** [comment]
Lgtm, thanks!

**Comment by mikedanese** [comment]
This doesn't remove the functions from pkg/util, it just copies them so that they are in two places. This also should be squashed.

**Comment by resouer** [comment]
@mikedanese Thanks for pointing out, just rebased

**Comment by mikedanese** [comment]
@resouer I think you should remove the functions from pkg/util you copied into pkg/util/strings to finish the migration. I don't think it's good to have duplicate functions in two places.

**Comment by mikedanese** [comment]
LGTM thanks!

**Pull request merged**